### ISSUE TITLE
Moodle Slack msg : Course Alert - Last week to complete

### ISSUE THREAD

**Issue by Goldy2488** [navgurukul/ghar#296]
slack message for progress alert that deadline is approached only 7 days/week is left.

**Comment by jyoti9909** [comment]
I prepare the msg format for the deadline reaches slack msg and apply the condition. 

today I will try to complete this message for slack. will co-ordinate to team.

**Comment by jyoti9909** [comment]
I had complete the Slack notification. And prepare report. During this we found bugs in code due to which we was unable to fetch the data. There was a wrong condition applied. 

So today will check the report because its not updated. So need to check. 
Check all reports and slack messages others are working fine.

**Comment by jyoti9909** [comment]
Yesterday we push the code and do some required changes and add comments and prepare code for Moodle tags and check the moodle modules to update the sheet for ghar criteria.

today willl work on report for weekly due date slack msg.

**Comment by jyoti9909** [comment]
Yesterday we set the report by using another field. We now able to track the report. 
We also handle that if curriculum progress is empty than manage learning page will also show null for same
Filter the unnecessary variable from the code
Ghar criteria missing is not found but we found that associate can put 0 in max min duration so we also prepare slack msg for that

Today will push this code and pick next issue whihc will handle the deletion scenerio on moodle.
